hack/lib/golang.sh

```diff
@@ -412,6 +412,10 @@ kube::golang::set_platform_envs() {
         export CGO_ENABLED=1
         export CC=${KUBE_LINUX_AMD64_CC:-x86_64-linux-gnu-gcc}
         ;;
+      "linux/386")
+        export CGO_ENABLED=1
+        export CC=${KUBE_LINUX_I686_CC:-i686-linux-gnu-gcc}
+        ;;
```

Comment:
We require a new kube-cross version for this, see: https://github.com/kubernetes/release/pull/2086